### PR TITLE
Develop react_tabs-with-router

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
[DEMO LINK](https://sergey-lipikhin.github.io/react_tabs-with-router/)

### CODE REVIEW

**Review by Anton-Tretiak — APPROVED** [review]
Well done!

File: src/common/heplers.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { tabs } from './tabs-data';
+
+export function getTabContentById(id: string | undefined): string | undefined {
+  return tabs
+    .find(tab => tab.id === id)
+    ?.content;
+}
```
Comment (OleksiiBlezniuk) [inline src/common/heplers.ts:7]: Better to find whole object intead, you may need other properties too

File: src/pages/TabsPage/TabsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { TabNav } from '../../components/TabNav';
+import { getTabContentById } from '../../common/heplers';
+
+export const TabsPage: React.FC = React.memo(() => {
+  const { selectedTabId = '' } = useParams();
```
Comment (OleksiiBlezniuk) [inline src/pages/TabsPage/TabsPage.tsx:7]: Better to create type for params and pass it as generic here

File: src/components/TabNav/TabNav.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { tabs } from '../../common/tabs-data';
+
+type Props = {
+  selectedTabId: string;
+};
+
+export const TabNav: React.FC<Props> = React.memo(({ selectedTabId }) => {
+  return (
```
Comment (OleksiiBlezniuk) [inline src/components/TabNav/TabNav.tsx:11]:
```suggestion
export const TabNav: React.FC<Props> = React.memo(({ selectedTabId }) => (
```

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**